setup-homebrew/main.sh

```diff
@@ -138,14 +168,16 @@ else
             git init
             git remote add origin "https://github.com/$GITHUB_REPOSITORY"
         fi
-        if [[ -z "${HOMEBREW_IN_CONTAINER-}" ]]; then
-            rm -rf "$GITHUB_WORKSPACE"
-            if [[ -n "${GITHUB_ACTIONS_HOMEBREW_SELF_HOSTED-}" ]]; then
-                mkdir -vp "$GITHUB_WORKSPACE"
-            else
-                ln -vs "$HOMEBREW_TAP_REPOSITORY" "$GITHUB_WORKSPACE"
-            fi
+
+        # Make repo available under `GITHUB_WORKSPACE` (default working directory), which some third-party taps may need.
+        # The symlink needs to be in this direction or `actions/cache` etc will break.
```

Comment:
```suggestion
        # The symlink needs to be in this direction or `actions/cache` etc. will break as they rely on `GITHUB_WORKSPACE` being `PWD`
```

or whatever is correct